cartridges/int_bolt_core/cartridge/scripts/utils/oauthUtils.js

```diff
@@ -103,7 +132,8 @@ function getOAuthConfiguration() {
  * @returns {Object} result
  */
 function exchangeOauthToken(code, scope, clientID, clientSecret, openIDConfig) {
-  var oauthTokenEndpoint = openIDConfig.token_endpoint;
+  //var oauthTokenEndpoint = openIDConfig.token_endpoint;
+  var oauthTokenEndpoint = "https://api.serena-external.dev.bolt.me/v1/oauth/token"
```

Comment:
I'm assuming this was accidentally left here.

Reply:
thanks! Removed test urls.